pkgs/tools/networking/nfstrace/default.nix

```diff
@@ -0,0 +1,37 @@
+{ cmake, fetchFromGitHub, fetchpatch, json_c, libpcap, ncurses, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "nfstrace";
+  version = "0.4.3.2";
+
+  src = fetchFromGitHub {
+    owner = "epam";
+    repo = "nfstrace";
+    rev = "${version}";
```

Comment:
```nix
    rev = version;
```
works too, you don't need to turn it into a string if you don't have a 'v' or something like this prefixing upstream tag.